test/test_nlp_expr.jl

```diff
@@ -371,26 +371,6 @@ function test_extension_recursion_stackoverflow(
     return
 end
 
-function test_nlparameter_interaction()
-    model = Model()
-    @variable(model, x)
-    @NLparameter(model, p == 1)
-    e = x + p
-    @test e isa GenericNonlinearExpr
-    @test string(e) == "x + ($p)"
-    return
-end
-
-function test_nlexpression_interaction()
-    model = Model()
-    @variable(model, x)
-    @NLexpression(model, expr, sin(x))
-    e = x + expr
```

Comment:
@pulsipher I'm not really sure what I intended to happen here.

Reply:
I am not sure, but I agree it should be removed in favor of this PR.